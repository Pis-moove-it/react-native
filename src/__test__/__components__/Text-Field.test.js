import React from 'react';
import renderer from 'react-test-renderer';
import TextField from '../../components/common/TextField';

test('Text Field renders correctly', () => {
  const tree = renderer.create(<TextField />).toJSON;
  expect(tree).toMatchSnapshot();
});
