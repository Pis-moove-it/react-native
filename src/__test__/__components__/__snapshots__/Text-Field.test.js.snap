// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Field renders correctly 1`] = `[Function]`;
